system-tests/lib/protocolStub.ts

```diff
@@ -1,5 +1,6 @@
 import type { ProtocolManagerShape } from '@packages/types'
 
+/*
```

Comment:
Are we planning on reimplementing the types here and they are just commented out temporarily?